docs/commerce/cart/cart.md

```diff
@@ -0,0 +1,97 @@
+---
+description: The Cart component covers adding items to the shopping cart, as well as previewing and modifying the cart information.
+edition: commerce
+---
+
+# Cart
+
+The Cart component is a foundation of the Commerce offering delivered as part 
+of [[= product_name =]].
+It covers actions related to the creation and handling of a list of products 
+that the buyer intends to purchase.
+
+
+The Cart component exposes the following:
+
+- [PHP API](cart_api.md) that allows for managing carts and cart entries, or validating products
+- [REST API](../../api/rest_api/rest_api_reference/rest_api_reference.html#managing-ecommerce-carts) that helps get cart and products information over HTTP
+- [Twig functions](../../templating/twig_function_reference/cart_twig_functions.md) that enable checking whether product can be added to cart and formatting the price
+
+There is no specific configuration related to the Cart component.
+All configuration is done at the Checkout and Storefront level.
+
+## Cart data handling
+
+Cart data is handled by two storages, depending on whether the buyer is anonymous 
+or has been authenticated.
+Information that relates to anonymous users is stored in the PHP session, while 
+registered user data is stored in a database.
+
+By default, anonymous users can add items to cart, but to display the cart view, 
```

Comment:
That may be technically the truth, but is confusing: can anonymous users even view products on the frontend by default?